sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/ScenarioTests/InboundEndpointTests.cs

```diff
@@ -0,0 +1,567 @@
+﻿// ------------------------------------------------------------------------------------------------
+// <copyright file="InboundEndpointTests.cs" company="Microsoft Corporation">
+//   Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------------------------
+
+namespace DnsResolver.Tests.ScenarioTests
+{
+    using FluentAssertions;
+    using Microsoft.Azure.Management.DnsResolver;
+    using Microsoft.Azure.Management.DnsResolver.Models;
+    using Microsoft.Azure.Management.DnsResolver.Tests.Extensions.Assertions;
+    using Microsoft.Azure.Management.Resources;
+    using Microsoft.Rest.Azure;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class InboundEndpointTests : BaseScenarioTests
+    {
+        public InboundEndpointTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void PutInboundEndpoint_EndpointNotExistsWithIpConfigurationsNoMetadata_ExpectEndpointCreated()
+        {
+            var resourceGroupName = this.CreateResourceGroup().Name;
+            var createdDnsResolver = this.CreateDnsResolver(resourceGroupName: resourceGroupName);
+            var inboundEndpointName = TestDataGenerator.GenerateInboundEndpointName();
+            var ipConfigurations = TestDataGenerator.GenerateRandomIpConfigurations(subscriptionId: this.SubscriptionId, resourceGroupName: resourceGroupName, virtualNetworkName: ExtractArmResourceName(createdDnsResolver.VirtualNetwork.Id));
+
+            var createdInboundEndpoint = this.DnsResolverManagementClient.InboundEndpoints.CreateOrUpdate(resourceGroupName: resourceGroupName, dnsResolverName: createdDnsResolver.Name, inboundEndpointName: inboundEndpointName, ipConfigurations: ipConfigurations);
+
+            createdInboundEndpoint.Should().BeSuccessfullyCreated();
```

Comment:
Don't you wanna have a check for properties of createdInboundEndpoint? Like VNet correctly associated, like a VNet GUID generated and associated, etc.?

Reply:
Are you refering to subnet provisioning related to  PUT IE? 
A new VNet is created during as part of PUT DnsResolver.

Reply:
Sorry, I thought I was looking at DNS resolver tests lol.